### PR TITLE
[IMP] Support branches in the environment variable VERSION of .travis.yml 

### PR DESCRIPTION
**Pull request by JesusZapata**
Support branches in the environment variable VERSION of .travis.yml

Now
```
VERSION=master
ODOO_REPO=odoo/odoo
```
Expected
```
VERSION=pull/3
ODOO_REPO=odoo/odoo
```

In the enviroment variable `VERSION` can be used:
- [x] Branch : VERSION="master"
- [x] PR : VERSION="pull/10

Referring to issue #173

### CODE REVIEW

File: travis/test_server.py [hunk 1]
```diff
@@ -136,8 +137,13 @@ def get_addons_path(travis_dependencies_dir, travis_build_dir, server_path):
     return addons_path
 
 
-def get_server_script(odoo_version):
-    return 'odoo-bin' if float(odoo_version) >= 10 else 'openerp-server'
+def get_server_script(server_path):
+    try:
+        shutil.copy(os.path.join(server_path, 'openerp-server'),
+                    os.path.join(server_path, 'odoo-bin'))
+    except IOError:
```
Comment (moylop260) [inline travis/test_server.py]: Add `# pragma: no cover` comment or add the correct test case from selftest

Comment (JesusZapata) [inline travis/test_server.py]: @moylop260 
You can help me?
Explained as run self_test ?

File: travis/self_tests [hunk 2]
```diff
@@ -80,6 +81,8 @@ if os.environ.get('LINT_CHECK', 0) == '1':
 # Testing git run from getaddons
 getaddons.get_modules_changed(repo_dir)
 
+# Testing get_server_script
+assert get_server_script("/tmp") == 'odoo-bin'
```
Comment (JesusZapata) [inline travis/self_tests]: @moylop260 
Is this the expected test?

**Comment by JesusZapata** [comment]
@moylop260 
In this PR now is also the solution to this issue https://github.com/Vauxoo/maintainer-quality-tools/issues/178

**Comment by JesusZapata** [comment]
@moylop260 
In this PR now is also the solution to this issue https://github.com/Vauxoo/maintainer-quality-tools/pull/181

File: travis/test_server.py (hunk 1 above)
Comment (dreispt) [inline travis/test_server.py]: It's best for the `get_server_script` to not have side effects.
You should instead check if `odoo-bin` exists, and fall back to `openep-server` if not.

Comment (JesusZapata) [inline travis/test_server.py]: @moylop260 
What do you think about the previous comment?
I do not remember where PR agreed this change

Comment (moylop260) [inline travis/test_server.py]: I'm agree with @dreispt 
(In fact, I don't remember the reason of use copy instead.)

Comment (JesusZapata) [inline travis/test_server.py]: @dreispt @moylop260 
I refactor the method `get_server_script` and now avoid the copy
I more simple

**Pull request merged**